LICENSE.txt

```diff
@@ -1,27 +1,35 @@
-Copyright (c) 2020, 2022 Oracle and/or its affiliates. All rights reserved.
+Copyright (c) 2020, 2022, 2023 Oracle and/or its affiliates.
```

Comment:
We usually just change the last year on copyright updates, e.g. 2020, 2023

Reply:
updated, thanks